dotnet/dotnet-cloud-run-hello-world/Pages/Index.cshtml

```diff
@@ -36,10 +36,10 @@
     </p>
 
     <h3>What's next?</h3>
-    <p class="next">Follow the Quickstart tutorial to build a “Hello World” application in your favorite
-        language into
-        a container image, and then deploy it to Cloud Run.</p>
+    <p class="next">Manage Cloud APIs and libraries in Cloud Code for <a href="https://cloud.google.com/code/docs/intellij/client-libraries" target="_blank" rel="noopener">IntelliJ</a> and <a href="https://cloud.google.com/code/docs/vscode/client-libraries" target="_blank" rel="noopener">VSCode</a></p>

```

Comment:
is it just arbitrary that we are directing to Cloud APIs for IntelliJ, and View Logs for VS Code?

also, do we want to get rid of VS Code references from this as well?

Reply:
We direct to Cloud APIs for both IntelliJ and VSCode.
As to Logs Viewer - I wanted to point to both extensions but then there was only logs viewer for VSCode.
We can hold VSCode references until they release Cloud Run to be consistent

Reply:
sgtm

Reply:
sgtm

Reply:
Updated the code

Reply:
etanshaul@, could you please LGTM so I can merge this?